quantum.bst

```diff
@@ -220,7 +219,12 @@ FUNCTION {no.output.check}
 FUNCTION {no.output.check.links}
 { 't :=
   duplicate$ empty$ nolink missing$ and
-    { pop$ "empty " t * ". If none of these fields is applicable for the cited work, please include the field `nolink = \{\}`, in the bib " * my.warning }
+    { pop$ "empty " t * ". " *
+      "If none of these fields is applicable for the cited work you can include the field " *
+%      newline$ *
+      "\verb|`nolink = {}`| in its bib" *
```

Comment:
I would like to use `\verb` here so that the correctly formatted code "nolink = {}," appears both in the LaTeX error message and the PDF ouput. The problem is that the whole thing `\verb|...|` must be on one line and BibTeX breaks the line automatically after a given number of characters, and where that break happens depends on thus on the length of the string in `t`. 
For example the resulting bbl can look like this:
```
\newblock Cambridge University PressCambridge, U.K.{\color{red}This bibitem
  caused a bibtex warning: empty doi, eprint and url. If none of these fields
  is applicable for the cited work you can include the field \verb|`nolink =
  {}`| in its
  bib}\PackageError{quantum.bst}{\quantumbsterror}{\quantumbsterror}
```
which does not compile correctly.

The solution would be to force BibTeX to start a new line in the generated bbl right before the `\verb` command. Unfortunately this cannot be achived by putting `newline$ *` as I tried in the commented line above.

@dwierichs, do you have an idea how to do that?

Reply:
I used LaTeX's `\newline` instead of BibTeX's `newline$` and it seems to work :)

Reply:
Unfortunately, `\newline` only works "coincidentally" because the whole sequence `\newline\verb|`nolink` is then long enough so that BibTeX decides to put it on a new line in this case. But if the length of `empty doi, eprint and url` changes, then this may break unexpectedly.

If I understand correctly, then the problem is that in this "postfix stack language" in which .bst files are written the "return value" of `no.output.check.links` is put on the stack and then poped in `check.links`. Somehow `newline$` does not survive this process. Is that correct? Maybe a solution could be to merge `no.output.check.links` and `check.links` into one function?

Reply:
I'm not sure I understand the problem that arises. Do you have a test case in which the `\newline` fails to produce the desired new line? It seems I can't produce one... :thinking:

Reply:
See here https://github.com/quantum-journal/quantum-journal/pull/130#issuecomment-1003764330